src/main/java/com/example/gateway/clients/ItemClient.java

```diff
@@ -0,0 +1,40 @@
+package com.example.gateway.clients;
+
+import com.example.lib.dto.create.ItemCreationRequest;
+import com.example.lib.dto.create.ItemCreationResponse;
+import com.example.lib.dto.read.ItemReadAllResponse;
+import com.example.lib.dto.read.ItemResponse;
+import com.example.lib.dto.update.ItemUpdateRequest;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.*;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.UUID;
+
+@FeignClient(value = "item", url = "${url.items}")
+public interface ItemClient {
```

Comment:
Этот клиент должен быть в item-service. И здесь браться из либы.

Reply:
Resolved. Before I had an exception of maven building because of I use in client libraries spring cloud dependencies. Now I drop it and use only spring-cloud-starter-openfeign. And it works.